apps/projection/src/app/ui/card/card.component.html

```diff
@@ -1,29 +1,18 @@
-<div class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3" [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px"
-  />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px"
-  />
-
+<div
+  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
+  [class]="customClass">
+  <ng-content></ng-content>
   <section>
     <app-list-item
-      *ngFor="let item of list"
-      [name]="item.firstname"
+      *ngFor="let item of items"
+      [name]="item.name"
       [id]="item.id"
-      [type]="type"
-    >
+      (_deleteItem)="_deleteItem.emit(item.id)">
     </app-list-item>
```

Comment:
What if I want to add an icon to my list-item? Or have a really custom item-list ?

Having only a name works for this very simple exemple, but we'll be very hard to highly customize in the future. 

You should look around ngTemplateOutlet

Reply:
I've added a ngTemplate inside `TeacherCardComponent`, `CityCardComponent`, and `StudentCardComponent` that will be used in `CardComponent` using content projection. Is this what you had in mind ?